kse/src/main/java/org/kse/crypto/x509/X509Ext.java

```diff
@@ -2231,21 +2202,21 @@ numeric INTEGER(1..999)
 
         if (currency != null) {
             String currencyString = currency.isAlphabetic() ? currency.getAlphabetic() : "" + currency.getNumeric();
-            sb.append(INDENT.toString(baseIndentLevel));
-            sb.append(MessageFormat.format(res.getString("QCEuLimitValue.Currency"), currencyString));
             sb.append(NEWLINE);
```

Comment:
Why is there now a NEWLINE before the INDENT?

Reply:
Hi @kaikramer 

Since I removed the `NEWLINE` from the line 2099.

Now, after all the tests, a newline is required before the indentations of:
- `if (currency != null)`
- `if (amount != null)`
- `if (exponent != null)`

And that's to ensure the same rule applies every time.

Because if I leave it on line 2099.
It will still need to be there for the following tests (hoping that `currency` isn't null, but it must be present):
- `if (amount != null)`
- `if (exponent != null)`

Therefore, to ensure a uniform rule, we add a NEWLINE before each indent in `getMonetaryValueStringValue`.

- Hope I'm clear?
- Don't hesitate to ask if you need more details.

Regards,
Th.